platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/StellarPaymentObserver.java

```diff
@@ -139,18 +153,33 @@ public void onEvent(OperationResponse operationResponse) {
             if (observedPayment != null) {
               try {
                 if (paymentObservingAccountsManager.lookupAndUpdate(observedPayment.getTo())) {
-                  final ObservedPayment finalObservedPayment = observedPayment;
-                  observers.forEach(observer -> observer.onReceived(finalObservedPayment));
+                  for (PaymentListener listener : observers) {
```

Comment:
What's the difference between the streaming API and the for-loop here? 😃

Reply:
It's because the `onReceived` method now throws an error and I wanted it to be catched by the outer try/catch.